### PR TITLE
Shilpa|BAH-631|Bahmni-package merging of bahmni-openmrs into bahmni-emr

### PR DESCRIPTION
**Pull request by shilpa-iplit**
https://bahmni.atlassian.net/browse/BAH-631

### CODE REVIEW

File: roles/bahmni-emr/tasks/main.yml [hunk 1]
```diff
@@ -7,18 +7,18 @@
...
 # Uninstall 2FA app before uninstalling OpenMRS
 - name: Uninstall two factor authentication rpm
   yum:  name="bahmni-two-factor-auth" state=absent
 
-- name: Uninstall bahmni-openmrs rpm
-  yum: name="bahmni-openmrs" state=absent
-  when: installed_openmrs.stdout.find("{{ bahmni_openmrs_version }}") == -1
+# - name: Uninstall bahmni-openmrs rpm
+  # yum: name="bahmni-openmrs" state=absent
+  # when: installed_openmrs.stdout.find("{{ bahmni_openmrs_version }}") == -1
```
Comment (SwathiVarkala) [inline roles/bahmni-emr/tasks/main.yml]: Why are we commenting instead of removing these lines? @shilpa-iplit

Comment (SwathiVarkala) [inline roles/bahmni-emr/tasks/main.yml]: After we merge bahmni-openmr, bahmni-emr rpms, do we still need this variable bahmni_openmrs_version? @shilpa-iplit

Comment (shilpa-iplit) [reply roles/bahmni-emr/tasks/main.yml]: I don't think variable bahmni_openmrs_version is needed, but need to check.
Yes will remove commented lines.

Comment (shilpa-iplit) [reply roles/bahmni-emr/tasks/main.yml]: Currently the variable bahmni_openmrs_version is defined in 2 files:
1) /bahmni-playbooks/group_vars/dev
2) /bahmni-playbooks/dev/rpm_versions.yml
Its not used anywhere. @SwathiVarkala

Comment (SwathiVarkala) [reply roles/bahmni-emr/tasks/main.yml]: @shilpa-iplit Please go ahead and remove the unused variable

Comment (shilpa-iplit) [reply roles/bahmni-emr/tasks/main.yml]: Removed the lines in updated PR.

Comment (shilpa-iplit) [reply roles/bahmni-emr/tasks/main.yml]: Removed the unused variable bahmni_openmrs_version

**Pull request merged**